oaknational/Oak-Web-Application
# Curric/lessons scrolling bug CUR-311

## Description

- Fixes scrolling bug on IOS

## Issue(s)

Fixes #CUR-311

## How to test

1. Go to https://deploy-preview-2032--oak-web-application.netlify.thenational.academy/teachers/curriculum/english-secondary-aqa/units on an iOS device.
2. Tap on a unit with a lot of lessons (e.g. `The Tempest`)
3. Open the `lessons in unit` modal and swipe to scroll up and down.
4. Scrolling should be smooth and easy no matter where your finger is located within the scrollable area.
5. Other parts of the page should be unaffected by this change.
6. Other browsers/device types should be unaffected by this change.

## Checklist
